### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by victyr**
This project is about a portfolio website using a reference of Figma design template. In this project, I have used HTML., CSS Flexbox, and JavaScript.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            <a href=""><i class="fa fa-angellist" aria-hidden="true"></i></a>
+          </li>
+          <li class="icon">
+            <a href=""><i class="fa fa-github" aria-hidden="true"></i></a>
+          </li>
+        </ul>
+      </div>
+      <div>
+        <button class="about-btn see_project" type="button">Get my resume</button>
+      </div>
```
Comment (vic778) [inline index.html:185]: - [ ] Kindly the font your are using and the color for social media are not match with the Figma template design.
- [ ] You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your design
Screenshot from 2022-05-28 16-09-01

FIgma design
Screenshot from 2022-05-28 16-09-27

**Review by vic778 — CHANGES_REQUESTED** [review]
Hello @victyr 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+        <p class="about-pitch pitch">
+          I'm a software developer! I can help you build a product, feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+      </div>
+      <div class="connect conn-about">
+        <h3>LETS CONNECT</h3>
+        <ul class="social-icons">
```
Comment (vic778) [inline index.html:164]: - [ ] Kindly select this ul in your css and add this code `font-size: x-large;
justify-content: space-evenly;
flex: fit-content;
margin-left: -30px; `they  gonna make your icons big and space between link this:
Screenshot from 2022-05-28 19-23-24

File: index.html [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+            <li><img src="images/css.png" alt=""/></li>
+            <li><img src="images/javaScript.png" alt="" /></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt=""/>
+        </div>
+      </div>
+    </section>
+    <section class="about-sect">
+      <div class="about">
+        <h2 class="about-intro intro">About <br />Myself</h2>
```
Comment (vic778) [inline index.html:154]: - [ ] Kindly select in you css file the h2 with through his `class` and ad this code `font-size: xx-large;` it gonna increase the size of the title.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+        <p class="about-pitch pitch">
+          I'm a software developer! I can help you build a product, feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
```
Comment (vic778) [inline index.html:158]: - [ ] Kindly select in you css file the `p` through his `class` and add this code `
font-size: large;` it going to increase the size of the paragraph.

**Review by vic778 — CHANGES_REQUESTED** [review]
Hello @victyr 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mke2111** [comment]
Project Under Second Opinion Review.
Kindly wait for a response. 😄

**Review by mke2111 — COMMENTED** [review]
Hello @victyr 👋,

_This is an additional review as a result of the second opinion request._
Great work with this project. 👍🏽

Kindly note that the requested changes should be optional since they were not included in the previous review. @vic778, always ensure to add all requests in the first review, and for all other re-reviews, request for extra changes as optional. This ensures that all reviews remain consistent.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Review by mukhammadrizooff — APPROVED** [review]
Hi @victyr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- All linters passed 💯 
- Correct use of Gitflow 💯 
- Descriptive messages 💯 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by victyr** [comment]
This is my full mobile version project

**Pull request merged**